setup.py

```diff
@@ -88,6 +88,7 @@ def run(self):
     # requirements files see:
     # https://packaging.python.org/en/latest/discussions/install-requires-vs-requirements/
     install_requires=[
+        "datasets>=2.7.0",
```

Comment:
Can we make this an optional dependency for now until we move Datalab out of experimental module?

Reply:
Also can we not have our package depend on matplotlib as a required dependency. Currently the `cleanlab.experimental.datalab.examples.data_generation.plot_data` function uses this, but let's just clarify in the experimental/README.md what auxiliary dependencies required for datalab and its examples are.

Reply:
`plot_data` is now only used in the tutorial notebook, which is now part of the docs.

Those already depend on matplotlib, so I think this is resolved.